PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackCuts.cxx

```diff
@@ -69,6 +70,12 @@ AliFemtoDreamTrackCuts::AliFemtoDreamTrackCuts()
       fNSigValueITS(3.),
       fPIDPTPCThreshold(0),
       fPIDPITSThreshold(0),
+      fTOFInMassCut(false),
+      fCutArroundPeakTOFInvMass(false),
+      fCutTOFInvMassSidebands(false),
+      fInvMassCutWidth(0),
+      fInvMassCutSBdown(0),
+      fInvMassCutSBup(0),
```

Comment:
please put fInvTOFMass... in all names in order to understand is related to TOF.